tests/e2e/driver_tests.py

```diff
@@ -630,6 +632,53 @@ def test_initial_namespace(self):
             cursor.execute("select current_database()")
             self.assertEqual(cursor.fetchone()[0], table_name)
 
+class PySQLStagingIngestionTestSuite(PySQLTestCase):
+    """Simple namespace for ingestion tests. These should be run against DBR >13.x
+    
+    In addition to connection credentials (host, path, token) this suite requires an env var
+    named staging_ingestion_user"""
+
+    staging_ingestion_user = os.getenv("staging_ingestion_user")
+
+    if staging_ingestion_user is None:
+        raise ValueError("To run these tests you must designate a `staging_ingestion_user` environment variable. This will the user associated with the personal access token.")
+
+    def test_staging_ingestion_put(self):
+
+        fh, temp_path =  tempfile.mkstemp()
+
+        with open(fh, 'wb') as fp:
+            fp.write("hello world!".encode("utf-8"))
+
+        with self.connection() as conn:
+            cursor = conn.cursor()
+            query = f"PUT '{temp_path}' INTO 'stage://tmp/{self.staging_ingestion_user}/tmp/11/15/file1.csv' OVERWRITE"
+            cursor.execute(query)
+
+        os.remove(temp_path)
```

Comment:
temp files created with `mkstemp()` must be manually removed per the Python docs